scripts/release/release_info.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import json
+import pathlib
+from typing import Dict
+
+from scripts.release.version import (
+    Environment,
+)
+
+
+class ImageInfo(dict):
+    def __init__(self, repository: str, platforms: list[str], version: str):
+        super().__init__()
+        self.repository = repository
+        self.platforms = platforms
+        self.version = version
+
+    def to_json(self):
+        return {"repository": self.repository, "platforms": self.platforms, "version": self.version}
+
+
+class BinaryInfo(dict):
+    def __init__(self, s3_store: str, platforms: list[str], version: str):
+        super().__init__()
+        self.s3_store = s3_store
+        self.platforms = platforms
+        self.version = version
+
+    def to_json(self):
+        return {"platforms": self.platforms, "version": self.version}
+
+
+class HelmChartInfo(dict):
+    def __init__(self, repository: str, version: str):
+        super().__init__()
+        self.repository = repository
+        self.version = version
+
+    def to_json(self):
+        return {"repository": self.repository, "version": self.version}
+
+
+class BuildInfo(dict):
+    def __init__(
+        self, images: Dict[str, ImageInfo], binaries: Dict[str, BinaryInfo], helm_charts: Dict[str, HelmChartInfo]
+    ):
+        super().__init__()
+        self.images = images
+        self.binaries = binaries
+        self.helm_charts = helm_charts
+
+    def __dict__(self):
+        return {
+            "images": {name: images.__dict__ for name, images in self.images.items()},
+            "binaries": {name: bin.__dict__ for name, bin in self.binaries.items()},
+            "helm-charts": {name: chart.__dict__ for name, chart in self.helm_charts.items()},
+        }
+
+    def to_json(self):
+        return {
+            "images": {name: images.to_json() for name, images in self.images.items()},
+            "binaries": {name: bin.to_json() for name, bin in self.binaries.items()},
+            "helm-charts": {name: chart.to_json() for name, chart in self.helm_charts.items()},
+        }
+
+
+def load_build_info(environment: Environment, version: str) -> BuildInfo:
```

Comment:
Should we add small comments to these functions? I think it would be helpful to mention, what should be the `version` value for `dev`, `staging` and `prod` etc?